projects/core/src/lib/colors/colorizer.model.ts

```diff
@@ -104,19 +123,23 @@ export class LinearGradient extends Colorizer {
         return new LinearGradient(
             this.breakpoints.map((b) => b.clone()),
             this.noDataColor.clone(),
-            this.defaultColor.clone(),
+            this.overColor.clone(),
+            this.underColor.clone(),
         );
     }
 
     cloneWith(updates: {
         readonly breakpoints?: Array<ColorBreakpoint>;
         readonly noDataColor?: Color;
-        readonly defaultColor?: Color;
+        readonly overColor?: Color;
+        readonly underColor?: Color;
     }): LinearGradient {
         return new LinearGradient(
             updates.breakpoints ?? this.breakpoints.map((b) => b.clone()),
             updates.noDataColor ?? this.noDataColor.clone(),
-            updates.defaultColor ?? this.defaultColor.clone(),
+            // TODO: refactor
```

Comment:
what is there to refactor?

Reply:
I already did this, the comment was a leftover:
[7b314a3](https://github.com/geo-engine/geoengine-ui/pull/573/commits/7b314a32ae65a87bca9de8d9427bdb979eddd26d)